reports/src/main/java/com/ivy/reports/ReportScreenEvent.kt

```diff
@@ -1,15 +1,16 @@
 package com.ivy.reports
 
 import android.content.Context
-import com.ivy.data.transaction.TransactionOld
+import android.net.Uri
 
 sealed class ReportScreenEvent {
```

Comment:
Remove `On` prefix from all events. 
Example `data class Filter(val filter: ReportFilter?) : ReportScreenEvent()` is way readable and events should mean action, not callback.